view/adminhtml/templates/info/payment_method.phtml

```diff
@@ -32,31 +32,46 @@ if ($giftCards = $block->getGiftCards()) {
                 <img src=" <?php echo $block->escapeHtml($block->getGiftcardLogo($giftCard)) ?>" alt=""/>
             </div>
             <div class="bk-label-wrap">
-            <?php echo $block->escapeHtml($giftCard['label']) ?>
+                <?php echo $block->escapeHtml($giftCard['label']) ?>
             </div>
         </div>
-    <?php
+        <?php
     }
+    ?>
+    <div class="bk-payment-wrap">
+        <div class="bk-img-wrap">
+            <img src="
+            <?php
+            $additionalInfo = $block->getGiftcardAdditionalData();
+            echo $block->escapeHtml($block->getPaymentLogo($additionalInfo['method_title'])) ?>
```

Comment:
why you did not do something like this,
    $additionalInfo = $block->getGiftcardAdditionalData();
    $methodTitle = $block->escapeHtml($additionalInfo['method_title']);
    $paymentLogoUrl = $block->escapeHtml($block->getPaymentLogo($methodTitle));
    ?>
    <div class="bk-payment-wrap">
        <div class="bk-img-wrap">
            <img src="<?ph echo  $paymentLogoUrl; ?>" alt="<?php echo $methodTitle; ?>" />
        </div>
        <div class="bk-label-wrap">
            <?ph echo $methodTitle; ?>
        </div>
    </div>
   it would be easier to be understandable I think